Looks like some test failures with simdj

```
ok  	github.com/minio/minio/internal/s3select/simdj	0.004s
sql.Missing{} (sql.Missing) != <nil> (<nil>)
--- FAIL: TestJsonpathEval (0.00s)
    jsonpath_test.go:93: case: 4 2 failed
FAIL
```

--- a/internal/s3select/select_test.go
+++ b/internal/s3select/select_test.go
@@ -556,6 +556,14 @@ func TestJSONQueries(t *testing.T) {
 			withJSON: `{"request":{"uri":"/1","header":{"User-Agent":"test"}}}
 {"request":{"uri":"/2","header":null}}`,
 		},
+		{
+			name:  "is-missing-select",
+			query: `select s.request.header['User-Agent'] as h from s3object[*] as s`,
+			wantResult: `{"h":"test"}
+{}`,
+			withJSON: `{"request":{"uri":"/1","header":{"User-Agent":"test"}}}
+{"request":{"uri":"/2","header":{}}}`,
+		},
 	}
 
 	defRequest := `<?xml version="1.0" encoding="UTF-8"?>

--- a/internal/s3select/sql/jsonpath_test.go
+++ b/internal/s3select/sql/jsonpath_test.go
@@ -66,32 +66,34 @@ func TestJsonpathEval(t *testing.T) {
 		{"s.authorInfo.yearRange", []interface{}{[]interface{}{1890.0, 1976.0}, []interface{}{1920.0, 1992.0}, []interface{}{1881.0, 1975.0}}},
 		{"s.authorInfo.name", []interface{}{"Agatha Christie", "Isaac Asimov", "P. G. Wodehouse"}},
 		{"s.authorInfo.yearRange[0]", []interface{}{1890.0, 1920.0, 1881.0}},
-		{"s.publicationHistory[0].pages", []interface{}{256.0, 336.0, nil}},
+		{"s.publicationHistory[0].pages", []interface{}{256.0, 336.0, Missing{}}},
 	}
 	for i, tc := range cases {
-		jp := JSONPath{}
-		err := p.ParseString(tc.str, &jp)
-		// fmt.Println(jp)
-		if err != nil {
-			t.Fatalf("parse failed!: %d %v %s", i, err, tc)
-		}
-
-		// Read only the first json object from the file
-		recs, err := getJSONStructs(b)
-		if err != nil || len(recs) != 3 {
-			t.Fatalf("%v or length was not 3", err)
-		}
-
-		for j, rec := range recs {
-			// fmt.Println(rec)
-			r, _, err := jsonpathEval(jp.PathExpr, rec)
+		t.Run(tc.str, func(t *testing.T) {
+			jp := JSONPath{}
+			err := p.ParseString(tc.str, &jp)
+			// fmt.Println(jp)
 			if err != nil {
-				t.Errorf("Error: %d %d %v", i, j, err)
+				t.Fatalf("parse failed!: %d %v %s", i, err, tc)
 			}
-			if !reflect.DeepEqual(r, tc.res[j]) {
-				fmt.Printf("%#v (%v) != %v (%v)\n", r, reflect.TypeOf(r), tc.res[j], reflect.TypeOf(tc.res[j]))
-				t.Errorf("case: %d %d failed", i, j)
+
+			// Read only the first json object from the file
+			recs, err := getJSONStructs(b)
+			if err != nil || len(recs) != 3 {
+				t.Fatalf("%v or length was not 3", err)
 			}
-		}
+
+			for j, rec := range recs {
+				// fmt.Println(rec)
+				r, _, err := jsonpathEval(jp.PathExpr, rec)
+				if err != nil {
+					t.Errorf("Error: %d %d %v", i, j, err)
+				}
+				if !reflect.DeepEqual(r, tc.res[j]) {
+					fmt.Printf("%#v (%v) != %v (%v)\n", r, reflect.TypeOf(r), tc.res[j], reflect.TypeOf(tc.res[j]))
+					t.Errorf("case: %d %d failed", i, j)
+				}
+			}
+		})
 	}
 }

